learn/_components/ExampleIcon.tsx

```diff
@@ -0,0 +1,19 @@
+export function ExampleIcon(props: { color: string }) {
+  return (
+    <svg
+      className="inline mr-2"
```

Comment:
Nit: I think it's better to style the SVG where it's used, rather than on the SVG itself. That way if we reuse the SVG elsewhere it doesn't come with a bunch of styling already in place to override. (Same note for the other two components.)